dipy/denoise/noise_estimate.py

```diff
@@ -1,9 +1,11 @@
 from __future__ import division, print_function
 
+import warnings
 import numpy as np
 
 from scipy.special import gammainccinv
 from scipy.ndimage.filters import convolve
+from scipy.stats import mode
```

Comment:
No - that's my bad. Wasn't sure what to do with this line in the rebase.
Keep looking, though. I might've messed up in other places...

On Mon, Feb 23, 2015 at 12:48 PM, Samuel St-Jean <notifications@github.com>
wrote:

> In dipy/denoise/noise_estimate.py
> <https://github.com/nipy/dipy/pull/576#discussion_r25200399>:
>
> >  import numpy as np
> >
> >  from scipy.special import gammainccinv
> >  from scipy.ndimage.filters import convolve
> > +from scipy.stats import mode
>
> the mode import is not used anymore, so maybe some other random stuff got
> in again :/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/576/files#r25200399>.
>